features/accommodations/creation/AmenitiesForm.tsx

```diff
@@ -0,0 +1,42 @@
+import { type FC } from "react";
+import { type Amenity } from "../AccommodationModels";
+import { type AmenitiesList, type CreateAccommodation } from "../createAccommodation";
+
+type AmenitiesFormProps = {
+  allAmenities: Amenity[];
+  updateFields: (fields: Partial<CreateAccommodation>) => void;
+  amenities: AmenitiesList;
+};
+
+export const AmenitiesForm: FC<AmenitiesFormProps> = ({
+  allAmenities,
+  updateFields,
+  amenities,
+}) => {
+  return (
+    <>
+      {allAmenities?.map((a) => (
+        <div className="flex space-x-2" key={a.id}>
+          <div>
+            <input
```

Comment:
SSimilar question about this checkbox into a component.

Reply:
We already have a component for a single checkbox, which couldn't be used here